### ISSUE TITLE
Input text and the generated voice do not match

### ISSUE THREAD

**Issue by wmathor** [NVIDIA/tacotron2#483]
First I used the dataset LJSpeech-1.1, I modified part of the structure of the Encoder (you can understand that I removed some convolutional layers or added several convolutional layers), `batch_size=42`, other parameters were not changed, and as a result I encountered the following problem during the inference. (ps: I have trained 61000 iters and the validation loss is 0.340894)
1. my input text is "waveglow is really awesome!", but the output speech is not this sentence, it is some other words into a sentence (although it is normal English speech)
2. alignment is empty, the model does not learn alignment information

![](https://user-images.githubusercontent.com/32392878/117524131-cf4ded00-afee-11eb-80f1-1ceccaaf7c47.jpg)

The audio file is here [https://soundcloud.com/athor-m/waveglow-is-really-awesome](https://soundcloud.com/athor-m/waveglow-is-really-awesome)

Do you have any advice for me? Thank you very much.

**Comment by khanifah-gif** [comment]
Hi @wmathor, have you finished this problem?
